aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -206,6 +235,23 @@ module aptos_std::simple_map {
         destroy_empty(map);
     }
 
+    #[test]
+    public fun test_add_all() {
+        let map = create<u64, u64>();
+
+        assert!(length(&map) == 0, 0);
+        add_all(&mut map, vector[1, 2, 3], vector[10, 20, 30]);
+        assert!(length(&map) ==3, 1);
+        assert!(borrow(&map, &1) == &10, 2);
+        assert!(borrow(&map, &2) == &20, 3);
+        assert!(borrow(&map, &3) == &30, 4);
+
+        remove(&mut map, &1);
+        remove(&mut map, &2);
+        remove(&mut map, &3);
+        destroy_empty(map);
+    }
+
```

Comment:
Nit: A test to assert the mismatching key / value lengths case results in an error would be good.